src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -2,30 +2,60 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonSetter;
 import com.google.common.collect.Lists;
 
 public class EventType {
 
     private String name;
+    private String owningApplication;
+    private String type;
     private final List<ValidationStrategyConfiguration> validationStrategies = Lists.newArrayList();
     private EventTypeSchema eventTypeSchema;
```

Comment:
I'm considering if we should accept things we don't actually use yet. Should I completely model EventType even if we are not using all the fields?